docs/examples/sock-shop.md

```diff
@@ -4,52 +4,83 @@ sidebar_position: 8
-✔ notification-controller: deployment ready
-✔ image-reflector-controller: deployment ready
 ✔ image-automation-controller: deployment ready
+✔ image-reflector-controller: deployment ready
+✔ kustomize-controller: deployment ready
+✔ notification-controller: deployment ready
+✔ source-controller: deployment ready
 ✔ install finished
+Deploy key generated and uploaded to git provider
+► Writing manifests to disk
+► Committing and pushing gitops updates for application
+► Pushing app changes to repository
+► Applying manifests to the cluster
 arete: /tmp/sock-shop>
 ```
 
-Once you see `install finished`, your cluster is ready to go with Weave GitOps.
+Once you see `► Applying manifests to the cluster`, your cluster is ready to go with Weave GitOps.
 
 ## Simple Deployment
 Once you have a cluster running Weave GitOps, it's simple to deploy an application like [*Sock Shop*](https://github.com/microservices-demo/microservices-demo).
 
-To deploy the *Sock Shop*, we need to use `gitops add app`. `gitops add app` comes in three flavors depending on how you'd like your GitOps support to be managed. Just as GitOps in general lets you synchronize your application definitions with their in-cluster versions via `git` operations and pull requests, Weave GitOps lets you manage the resources that _do_ the synchronization in the same way. That might sound a bit "meta" but it allows you to update the management resources themselves via pull requests. You can change things like synchronization intervals or even upgrade the resources via git. Why _wouldn't_ you want to be able to do those things?
-
-The three storage options for the management resources provided by Weave GitOps are:
-- in a `.wego` subdirectory of the same repository as your application (for a simple all-in-one configuration)
-- in a separate configuration repository (allowing you to collect all your management resources together)
-- _only_ in the cluster (this is mostly intended for quick turnaround and testing)
+To deploy the *Sock Shop*, we need to use `gitops add app`. `gitops add app` will store the GitOps automation support for your application in the `.weave-gitops` directory of the configuration repository you specified at install time. The definition of your application can be stored either in a separate repository or in the configuration repository itself (for a simple all-in-one configuration). If you want to store the application resources in the configuration repository, you only need to specify the `--url` flag which will be used for both application and configuration resources; however, this assumes that the application repository URL was passed to `gitops install`. If you want the application resources to be stored separately, you need to specify both `--url` and `--app-config-url` parameters. The `--url` parameter should be the URL of the repository containing the application definition and the `--app-config-url` parameter should be the URL that was used in `gitops install`.
```

Comment:
I think we should be clear that add _requires_ the app-config-url parameter to match the repo used in install.  If the user doesn't specify app-config-url on add, we _assume_ that install was run with the application repo.  When I read this it sounds like we will figure out the install repo and use that - which we should consider adding as that will be helpful for users.

I see you call this out below which is good but we should also call it out here.